packages/terra-navigation-side-menu/src/MenuItem.module.scss

```diff
@@ -15,7 +15,7 @@
     color: var(--terra-navigation-side-menu-item-color, #1c1f21);
     cursor: pointer;
     display: flex;
-    font-size: var(--terra-navigation-side-menu-item-size, 1.071rem);
+    font-size: var(--terra-navigation-side-menu-item-font-size, 1.071rem);
     line-height: var(--terra-navigation-side-menu-item-line-height, 1.42857);
```

Comment:
@yuderekyu Is this non-passive?

Reply:
yeah, i'd say revert this change until we major version bump this with the rest of the refactored theme variables

Reply:
Reverted this changes.